db_patches/0114_AddLocalContactToScheduledEvents.sql

```
@@ -0,0 +1,15 @@
+DO
+$DO$
+BEGIN
+  IF register_patch('AddLocalContactToScheduledEvents.sql', 'Martin Trajanovski', 'Add local contact person to the scheduled event', '2021-12-08') THEN
+  BEGIN
+
+    ALTER TABLE "scheduled_events"
+    ADD COLUMN IF NOT EXISTS "local_contact" int DEFAULT NULL
+    REFERENCES users(user_id) ON DELETE CASCADE;
```

I would suggest to not have here `ON DELETE CASCADE`. 
If someone tries to delete the user, let it fail instead of automatically removing scheduled events to whom the user is local contact.

Alternatively set the local_contact to null when deleting which also would work, as the default value is null already
`REFERENCES users(user_id) ON DELETE SET NULL`